public/locales/en/translation.json

```diff
@@ -2,6 +2,9 @@
   "Home": "Home",
   "Shop": "Shop",
   "Account": "Account",
+  "New Arrivals": "New Arrivals",
+  "Hot Products": "Hot Products",
+  "Check out our new furniture collection! Cozy sofa, fancy chair, wooden casket, and many more. The new collection brings an informal elegance to your home.": "Check out our new furniture collection! Cozy sofa, fancy chair, wooden casket, and many more. The new collection brings an informal elegance to your home.",
```

Comment:
key ngắn thôi e, cần gì viết cả đoạn thế này. VD: "check out our new furniture collection": "Check out our new furniture collection! Cozy sofa, fancy chair, wooden casket, and many more. The new collection brings an informal elegance to your home."